package.json

```diff
@@ -193,15 +195,17 @@
 		"grunt-checktextdomain": "1.0.1",
 		"grunt-wp-i18n": "1.0.3",
 		"husky": "4.3.8",
-		"jest-24.9.0": "npm:jest@24.9.0",
+		"jest": "^25.5.4",
```

Comment:
No need for a specific locked version like the above (from what I could see).

Reply:
This was done by me in a previous issue to address a problem with jest performance. Running jest locally has really bad performance in 25.x. See https://github.com/facebook/jest/issues/9457

That said, I think things have been getting better in 26... I wonder if we could look at using 26 and see how it goes?

Reply:
Gotcha, I will revert this then, that is also rather unfortunate :( 
I saw people still reproducing it in 26 (https://github.com/facebook/jest/issues/9457#issuecomment-736024084) so I will revert this.

Reply:
I might add a comment though, as an FYI